app/components/layout/install-pwa-prompt-modal.tsx

```diff
@@ -0,0 +1,71 @@
+import { useLoaderData, useFetcher } from "@remix-run/react";
+import { AnimatePresence } from "framer-motion";
+
+import type { loader } from "~/routes/_layout+/_layout";
+import { usePwaManager } from "~/utils/pwa-manager";
+import { Button } from "../shared/button";
+import { useRef } from "react";
+
+export function InstallPwaPromptModal() {
+  const { hideInstallPwaPrompt } = useLoaderData<typeof loader>();
+  const fetcher = useFetcher();
+  let optimisticHideInstallPwaPrompt = hideInstallPwaPrompt;
+  if (fetcher.formData) {
+    optimisticHideInstallPwaPrompt =
+      fetcher.formData.get("pwaPromptVisibility") === "hidden";
+  }
+  const hidePwaPromptForm = useRef<HTMLFormElement | null>(null);
+
+  const { promptInstall } = usePwaManager();
+  return optimisticHideInstallPwaPrompt ? null : (
+    <AnimatePresence>
+      <div className="dialog-backdrop !items-end !bg-[#364054]/70">
+        <dialog
+          className="dialog m-auto h-auto w-[90%] pb-8 sm:w-[400px]"
+          open={true}
+        >
+          <div className="relative z-10  rounded-xl bg-white p-4 shadow-lg">
+            <video height="200" autoPlay loop muted className="mb-6 rounded-lg">
```

Comment:
This doesn't auto play on my iphone. I think you need to add the inline attribute for it to work.